ynh-dev

```diff
@@ -145,6 +147,21 @@ elif [ "$1" = "run" ]; then
 
 
 #####################
+## Halt running VM ##
+#####################
+elif [ "$1" = "halt" ]; then
+    vm=$(vagrant global-status | head -3 | tail -1)
```

Comment:
Maye by we can mimic what the `ynh-dev run` command do, by adding an parameter with version name. That way we can grep the output of `vagrant global-status`